lms/static/js/student_profile/views/learner_profile_factory.js

```diff
@@ -40,7 +40,9 @@
                 required: true,
                 editable: 'always',
                 showMessages: false,
-                title: gettext('edX learners can see my:'),
+                title: interpolate_text(
+                    gettext('{platform_name} learners can see my:'), {platform_name: options.platform_name}
```

Comment:
@srpearce @explorerleslie - I wonder if this will not work so well with all possible platform names. It also strikes me as being a little problematic for right-to-left languages.